build/README.md

```diff
@@ -1,60 +1,170 @@
+# Start with docker compose
+
+You can start ZITADEL with a simple docker compose up.
+
+The services are configured to restart if an error occurs. Until the essential services are started and executed some services panic, this is expected just give it some minutes to setup the database and execute migrations.
+
+```bash
+COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 docker compose -f ./build/local/docker-compose-dev.yml --profile init --profile backend --profile frontend -p zitadel up
+```
 
 # Development
 
+You should stay in the ZITADEL root directory to execute the statements in the following chapters.
+
 ## Prerequisite
 
 - Buildkit compatible docker installation
 
-## Generate Proto Clients
+### env variables
+
+Default env variables are provided in [this .env-file](build/local/local.env)
```

Comment:
Does the user need to manipulate these or can the "just be used"?

Reply:
in default use cases it can "just be used"

Reply:
In this case should state this more clearly. Like `You can use the default vars provided ... or create your own ... `

Reply:
changed text